core/xml/src/test/java/org/opennms/core/xml/NumberAdapterTest.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2019 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2019 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.core.xml;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class NumberAdapterTest {
+
+    @Test
+    public void testNumberAdapter() throws Exception {
+
+        NumberAdapter adapter = new NumberAdapter();
+        String value = adapter.marshal(4.65450971582E11);
+        assertTrue(value.equals("4.65450971582E11"));
```

Comment:
We should use `assertEquals` instead of `assertTrue` here.

Can we also verify that unmarshalling the resulting string gives us a similar result?